compareECLFiles.cmake

```diff
@@ -962,6 +962,14 @@ add_test_compareECLFiles(CASENAME actionx_wefac
                          REL_TOL ${rel_tol}
                          DIR actionx)
 
+add_test_compareECLFiles(CASENAME 0-BASE-MODEL6
+                         FILENAME 0_BASE_MODEL6
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR model6)
```

Comment:
Here I would suggest also invoking the restart test functionality by adding a line
```
    ....
    REL_TOL ${rel_tol}
    RESTART_STEP 10
    DIR model6)
```

This will add a restart test starting from step 10 (whether that numerical value makes sense I do not know).